testing/dart/window_hooks_integration_test.dart

```diff
@@ -125,65 +137,99 @@ void main() {
     test('onPointerDataPacket preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      PointerDataPacket data;
 
       runZoned(() {
         innerZone = Zone.current;
-        window.onPointerDataPacket = (_) {
+        window.onPointerDataPacket = (PointerDataPacket value) {
           runZone = Zone.current;
+          data = value;
         };
       });
 
-      _dispatchPointerDataPacket(new ByteData.view(new Uint8List(0).buffer));
+      final ByteData testData = new ByteData.view(new Uint8List(0).buffer);
+      _dispatchPointerDataPacket(testData);
       expect(runZone, isNotNull);
       expect(runZone, same(innerZone));
+      expect(data.data, equals(_unpackPointerDataPacket(testData).data));
     });
 
     test('onSemanticsEnabledChanged preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      bool enabled;
 
       runZoned(() {
         innerZone = Zone.current;
         window.onSemanticsEnabledChanged = () {
           runZone = Zone.current;
+          enabled = window.semanticsEnabled;
         };
       });
 
       _updateSemanticsEnabled(window._semanticsEnabled);
       expect(runZone, isNotNull);
       expect(runZone, same(innerZone));
+      expect(enabled, isNotNull);
+      expect(enabled, equals(window._semanticsEnabled));
     });
 
     test('onSemanticsAction preserves callback zone', () {
       Zone innerZone;
       Zone runZone;
+      int action;
 
       runZoned(() {
         innerZone = Zone.current;
-        window.onSemanticsAction = (_, __) {
+        window.onSemanticsAction = (int value, _) {
```

Comment:
also, not a big deal, but as a general rule we should always include types (even for unused arguments), that way when someone is looking at the code it's more self-documenting, easier to copy/paste, etc. Don't worry about adding the type (and maybe explicit variable name) here unless you happy to be on this line some time, though.

Reply:
I think for tests that's less of an issue.  I think it adds to readability of the test to be able to just visually skip over unused arguments (because they are so often irrelevant).  I agree that for framework code this is reversed, and that they should always have types.